integration-tests/conftest.py

```diff
@@ -1,10 +1,27 @@
 import pytest
+import subprocess
 
 
+@pytest.fixture(scope="session")
+def install_katello_rpm(test_config):
+    if "satellite" in test_config.environment:
+        # install katello rpm before register system against Satellite
+        cmd = "rpm -qa 'katello*' | xargs --no-run-if-empty rpm -e"
+        ret = subprocess.check_call(cmd, shell=True)
+        
+        satellite_hostname = test_config.get("candlepin", "host")
+        cmd = "rpm -Uvh http://%s/pub/katello-ca-consumer-latest.noarch.rpm" %satellite_hostname
+        ret = subprocess.check_call(cmd, shell=True)
```

Comment:
`shell=True` is not needed for this command, as it is a simple command invocation